hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleContextImpl.java

```diff
@@ -578,6 +584,13 @@ private void dispatchSyntheticTxn(
                 return;
             }
         }
+
+        final var validationResult = validate(verifier, function, txBody, payer, payerKey);
```

Comment:
Are you sure these are the right parameters? I would have expected `childVerifier`, `syntheticPayer`, and `childPayerKey`.